core/src/main/kotlin/io/saagie/updatarium/dsl/ChansetSet.kt

```diff
@@ -53,11 +53,13 @@ data class ChangeSet(
                     it.execute()
                 }
                 InMemoryAppenderManager.stopRecord()
-                persistEngine.unlock(this, Status.OK,InMemoryAppenderAccess.getEvents())
+                persistEngine.unlock(this, Status.OK, InMemoryAppenderAccess
+                    .getEvents(persistConfig = persistEngine.configuration,success = true))
                 logger.info { "$id marked as ${Status.OK}" }
             } catch (e: Exception) {
                 logger.error(e) { "Error during apply update" }
-                persistEngine.unlock(this, Status.KO, InMemoryAppenderAccess.getEvents())
+                persistEngine.unlock(this, Status.KO, InMemoryAppenderAccess
+                    .getEvents(persistConfig = persistEngine.configuration,success = false))
```

Comment:
missing space between `persistEngine.configuration` and `success = false`

Reply:
LinterMan !

Reply:
Fixed